src/Util/generateUUID.ts

```diff
@@ -0,0 +1,8 @@
+const generateUUID = () => {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+    var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);

```

Comment:
It seems to be less chances of having a duplicate id but it's ok, I'm fine with using bitwise despite I don't find a big benefit of using it here, just needs some cleaning to be kind of this:

```js
const randomNum = Math.random() * 16 | 0
const finalChar = c === 'x' ? randomNum : (randomNum & 0x3 | 0x8);
```